ansible/roles/internal/jupyterhub/templates/jupyter-http.conf.j2

```diff
@@ -62,22 +62,6 @@ RequestHeader unset REMOTE_USER
   </Location>
   {% endif %}
 
-{% if hub_metrics == true %}
```

Comment:
Don't we need something like this to proxy connections through to node-exporter or cadvisor?

Reply:
The work that was done a few months ago leveraged apache to do the routing, but now we have Caddy to do the routing.